### PR TITLE
Some boolean props not working

### PR DESCRIPTION
**Pull request by ashwinKumar0505**

### CODE REVIEW

File: config-overrides.js [hunk 1]
```diff
@@ -1,8 +1,5 @@
-const { override, addBundleVisualizer } = require('customize-cra')
+const { override } = require('customize-cra')
 const { addReactRefresh } = require('customize-cra-react-refresh')
 
 /* config-overrides.js */
-module.exports = override(
-  addReactRefresh({ disableRefreshCheck: true }),
```
Comment (sanandnarayan) [inline config-overrides.js:6]: why are we disabling refresh?

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@bugsnag/js": "^6.5.2",
     "@bugsnag/plugin-react": "^6.5.0",
-    "@chakra-ui/core": "next",
```
Comment (sanandnarayan) [inline package.json:8]: is next the latest version?

**Review by sanandnarayan — APPROVED** [review]
Hi i made a couple of comments, can you respond and merge?

File: package.json (hunk 2 above)
Comment (ashwinKumar0505) [reply package.json:8]: No, 1.0.0-rc.5 is the latest version

File: config-overrides.js (hunk 1 above)
Comment (ashwinKumar0505) [reply config-overrides.js:6]: I am not disabling it bro. I am adding react- refresh to the project. I don't know why they had disabled the refreshCheck by the openchakra,  I had removed it. The configuration setup is done with the help of https://github.com/esetnik/customize-cra-react-refresh

**Pull request merged**